frontend/components/ModelForm.tsx

```diff
@@ -93,6 +94,7 @@ type ModelTableProps = {
   objects: ModelObject[]
   allowEditing?: boolean
   allowDeletion?: boolean
+  minimumRequired?: number
```

Comment:
@alnasir7 Remove this variable if you're not using it anymore!

Reply:
@alnasir7 Reopening this because it seems like this still exists without being used.